reference/tax.v3.yml

```diff
@@ -108,11 +108,16 @@ paths:
                 password:
                   type: string
                   example: h6eSgKLN72q7jYTW
+                profile:
+                  type: string
+                  description: This is an optional field and will be used with supporting providers only.
```

Comment:
I am not sure this gives me much extra information really. 
I know that the profile enables flexibility in urls but I would not be able to discern that from the above text.